gno.land/cmd/gnoland/start.go

```diff
@@ -235,6 +235,10 @@ func makeGenesisDoc(
 	for _, pkg := range nonDraftPkgs {
 		// open files in directory as MemPackage.
 		memPkg := gno.ReadMemPackage(pkg.Dir, pkg.Name)
+		if len(memPkg.Files) == 0 { // skip empty package
```

Comment:
is this case possible @harry-hov?
I think your helper should skip this by default.